web/src/demo_components/list_group_item.dart

```diff
@@ -0,0 +1,241 @@
+  /// representation when rendered.
+  ///
+  /// Will only be used if [href] and [onClick] are both `null`.
+  ///
+  /// Default: [ListGroupItemElementType.SPAN]
+  ListGroupItemElementType elementType;
+
+  /// Optional header text to display within the [ListGroupItem] above
+  /// the value of [children].
+  ///
+  /// See: <http://v4-alpha.getbootstrap.com/components/list-group/#custom-content>.
+  dynamic header;
+
+  /// The size of the [header] text you desire.
+  ///
+  /// Default: [ListGroupItemHeaderElementSize.H5]
+  ListGroupItemHeaderElementSize headerSize;
+
+  /// Additional props to be added to the [header] element _(if specified)_.
+  Map headerProps;
+
+  /// The skin / "context" for the [ListGroupItem].
+  ///
+  /// See: <http://v4-alpha.getbootstrap.com/components/list-group/#contextual-classes>.
+  ///
+  /// Default: [ListGroupItemSkin.DEFAULT]
+  ListGroupItemSkin skin;
+
+  /// Whether the [ListGroupItem] should appear "active".
+  ///
+  /// See: <http://v4-alpha.getbootstrap.com/components/list-group/#anchors-and-buttons>
+  ///
+  /// Default: false
+  bool isActive;
+
+  /// Whether the [ListGroupItem] is disabled.
+  ///
+  /// See: <http://v4-alpha.getbootstrap.com/components/list-group/#disabled-items>
+  ///
+  /// Default: false
+  @Accessor(key: 'disabled', keyNamespace: '')
+  bool isDisabled;
+
+  /// The HTML `href` attribute value for the [ListGroupItem].
+  ///
+  /// If set, the item will render via [Dom.a].
+  ///
+  /// _Proxies [DomProps.href]_
+  @Accessor(keyNamespace: '')
+  String href;
+
+  /// The HTML `target` attribute value for the [ListGroupItem].
+  ///
+  /// If set, the item will render via [Dom.a].
+  ///
+  /// _Proxies [DomProps.target]_
+  @Accessor(keyNamespace: '')
+  String target;
+
+  /// The HTML `type` attribute value for the [ListGroupItem] when
+  /// rendered via [Dom.button].
+  ///
+  /// This will only be applied if [onClick] is also set.
+  ///
+  /// _Proxies [DomProps.type]_
+  ///
+  /// Default: [ButtonType.BUTTON]
+  ButtonType type;
+}
+
+@Component()
+class ListGroupItemComponent extends UiComponent<ListGroupItemProps> {
+  @override
+  Map getDefaultProps() => (newProps()
+    ..elementType = ListGroupItemElementType.SPAN
+    ..skin = ListGroupItemSkin.DEFAULT
+    ..isActive = false
+    ..isDisabled = false
+    ..type = ButtonType.BUTTON
+    ..headerSize = ListGroupItemHeaderElementSize.H5
+  );
+
+  @override
+  render() {
+    var children = props.children;
+
+    if (props.header != null) {
+      children = [
+        renderItemHeader(),
+        (Dom.p()
+          ..className = 'list-group-item-text'
+          ..key = 'item-text'
+        )(props.children)
+      ];
+    }
+
+    BuilderOnlyUiFactory<DomProps> factory = _getItemDomNodeFactory();
+
+    return (factory()
+      ..addProps(copyUnconsumedDomProps())
+      ..className = _getItemClasses().toClassName()
+      ..href = props.href
```

Comment:
Won't work since they are within `ListGroupItemProps` - which is consumed by default.